src/kontrol/kdist/cheatcodes.md

```diff
@@ -1328,7 +1328,7 @@ Will also return true if REASON is `.Bytes`.
            <prevOrigin> _ => OG </prevOrigin>
            <newCaller> _ => NEWCALLER </newCaller>
            <newOrigin> _ => NEWORIGIN </newOrigin>
-           <active> false => true </active>
+           <active> _ => true </active>
```

Comment:
Yes, so this will allow the next prank to override the previous one rather than not fire.

Reply:
Yes. Currently it gets stuck.